ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean] Failed to initialize JPA EntityManagerFactory: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275): Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.engine.jdbc.dialect.internal.DialectFactoryImpl.determineDialect(DialectFactoryImpl.java:100): Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean] Failed to initialize JPA EntityManagerFactory: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275): Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.engine.jdbc.dialect.internal.DialectFactoryImpl.determineDialect(DialectFactoryImpl.java:100): Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean] Failed to initialize JPA EntityManagerFactory: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:421): [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateTable(AbstractSchemaValidator.java:121): Schema-validation: missing table [administrator]
ERROR [com.zaxxer.hikari.pool.HikariPool] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
WARN [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120): Access denied for user 'acme-user'@'%' to database 'starter-project'
ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean] Failed to initialize JPA EntityManagerFactory: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275): Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
at org.hibernate.engine.jdbc.dialect.internal.DialectFactoryImpl.determineDialect(DialectFactoryImpl.java:100): Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean] Failed to initialize JPA EntityManagerFactory: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
WARN [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
ERROR [org.springframework.boot.SpringApplication] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:421): [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [administrator]
at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateTable(AbstractSchemaValidator.java:121): Schema-validation: missing table [administrator]
INFO [acme.Launcher] Starting Launcher using Java 1.8.0_281 on LAPTOP-8V5BH0HP with PID 17092 (C:\Users\lenovo\Desktop\Ingeniería Software\3º\Segundo Cuatri\DP2\Workspace-21.1\Workspace-21.1\Starters\Starter-Project\target\classes started by lenovo in C:\Users\lenovo\Desktop\Ingeniería Software\3º\Segundo Cuatri\DP2\Workspace-21.1\Workspace-21.1\Starters\Starter-Project)
INFO [acme.Launcher] The following profiles are active: development,populator
INFO [acme.Launcher] Started Launcher in 19.774 seconds (JVM running for 21.802)
INFO [acme.framework.utilities.DatabasePopulator] Populating initial data
INFO [acme.framework.utilities.DatabasePopulator] Recreating schema
INFO [acme.framework.utilities.DatabasePopulator] Populating entities from classpath:/WEB-INF/population/initial-data.xml
INFO [acme.framework.utilities.DatabasePopulator] Validating your entities.
INFO [acme.framework.utilities.DatabasePopulator] Validating anonymousUserAccount ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating anonymousAnonymous ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorUserAccount ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorAdministrator ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorAuthenticated ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Sorting your entities topologically.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 0 for anonymousUserAccount.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 1 for anonymousAnonymous.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 2 for administratorUserAccount.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 3 for administratorAdministrator.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 4 for administratorAuthenticated.
INFO [acme.framework.utilities.DatabasePopulator] Persiting your entities.
INFO [acme.framework.utilities.DatabasePopulator] > Persisting anonymousUserAccount = acme.framework.entities.UserAccount{id=1, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting anonymousAnonymous = acme.framework.entities.Anonymous{id=2, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorUserAccount = acme.framework.entities.UserAccount{id=3, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorAdministrator = acme.framework.entities.Administrator{id=4, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorAuthenticated = acme.framework.entities.Authenticated{id=5, version=0}
INFO [acme.framework.configuration.ShutdownConfiguration] The system is shutting down...
INFO [acme.Launcher] Starting Launcher using Java 1.8.0_281 on LAPTOP-8V5BH0HP with PID 9800 (C:\Users\lenovo\Desktop\Ingeniería Software\3º\Segundo Cuatri\DP2\Workspace-21.1\Workspace-21.1\Starters\Starter-Project\target\classes started by lenovo in C:\Users\lenovo\Desktop\Ingeniería Software\3º\Segundo Cuatri\DP2\Workspace-21.1\Workspace-21.1\Starters\Starter-Project)
INFO [acme.Launcher] The following profiles are active: development,populator
INFO [acme.Launcher] Started Launcher in 18.475 seconds (JVM running for 21.151)
INFO [acme.framework.utilities.DatabasePopulator] Populating initial data
INFO [acme.framework.utilities.DatabasePopulator] Recreating schema
INFO [acme.framework.utilities.DatabasePopulator] Populating entities from classpath:/WEB-INF/population/initial-data.xml
INFO [acme.framework.utilities.DatabasePopulator] Validating your entities.
INFO [acme.framework.utilities.DatabasePopulator] Validating anonymousUserAccount ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating anonymousAnonymous ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorUserAccount ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorAdministrator ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Validating administratorAuthenticated ... PASS
INFO [acme.framework.utilities.DatabasePopulator] Sorting your entities topologically.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 0 for anonymousUserAccount.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 1 for anonymousAnonymous.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 2 for administratorUserAccount.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 3 for administratorAdministrator.
INFO [acme.framework.utilities.DatabasePopulator] > Setting index 4 for administratorAuthenticated.
INFO [acme.framework.utilities.DatabasePopulator] Persiting your entities.
INFO [acme.framework.utilities.DatabasePopulator] > Persisting anonymousUserAccount = acme.framework.entities.UserAccount{id=1, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting anonymousAnonymous = acme.framework.entities.Anonymous{id=2, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorUserAccount = acme.framework.entities.UserAccount{id=3, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorAdministrator = acme.framework.entities.Administrator{id=4, version=0}
INFO [acme.framework.utilities.DatabasePopulator] > Persisting administratorAuthenticated = acme.framework.entities.Authenticated{id=5, version=0}
INFO [acme.framework.configuration.ShutdownConfiguration] The system is shutting down...
WARN [org.hibernate.orm.connections.pooling] HHH10001002: Using Hibernate built-in connection pool (not for production use!)
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [com.zaxxer.hikari.pool.PoolBase] {} - Failed to validate connection {} ({}). Possibly consider using a shorter maxLifetime value.
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30001ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] HTTP POST /Starter-Project/WEB-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Exception: org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467): Could not open JPA EntityManager for transaction; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48): Unable to acquire JDBC Connection
com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:695): HikariPool-1 - Connection is not available, request timed out after 30001ms.
com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30000ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[jsp]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[dispatcherServlet]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] HTTP POST /Starter-Project/master/panic HTTP/1.1
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Exception: org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping.handleNoMatch(RequestMappingInfoHandlerMapping.java:250): Request method 'POST' not supported
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30000ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[jsp]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[dispatcherServlet]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] HTTP GET /Starter-Project/master/panic HTTP/1.1
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Exception: org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014): Request processing failed; nested exception is org.apache.tiles.request.render.CannotRenderException: ServletException including path '/WEB-INF/views/framework/page.jsp'.
org.apache.tiles.impl.BasicTilesContainer.render(BasicTilesContainer.java:399): ServletException including path '/WEB-INF/views/framework/page.jsp'.
org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): ServletException including path '/WEB-INF/views/framework/page.jsp'.
org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/debug-principal.tag(19,1) '${PrincipalHelper.get()}' org.springframework.dao.DataAccessResourceFailureException: Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:255): Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48): Unable to acquire JDBC Connection
com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:695): HikariPool-1 - Connection is not available, request timed out after 30000ms.
com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
sun.reflect.GeneratedConstructorAccessor70.newInstance(Unknown Source): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30000ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[jsp]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[dispatcherServlet]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost]] at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014): Request processing failed; nested exception is org.apache.tiles.request.render.CannotRenderException: ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.tiles.impl.BasicTilesContainer.render(BasicTilesContainer.java:399): ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/debug-principal.tag(19,1) '${PrincipalHelper.get()}' org.springframework.dao.DataAccessResourceFailureException: Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:255): Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48): Unable to acquire JDBC Connection
at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:695): HikariPool-1 - Connection is not available, request timed out after 30000ms.
at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
at sun.reflect.GeneratedConstructorAccessor70.newInstance(Unknown Source): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30000ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[jsp]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/Starter-Project].[dispatcherServlet]] at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
ERROR [org.apache.catalina.core.ContainerBase.[Tomcat].[localhost]] at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014): Request processing failed; nested exception is org.apache.tiles.request.render.CannotRenderException: ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.tiles.impl.BasicTilesContainer.render(BasicTilesContainer.java:399): ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.tiles.request.servlet.ServletUtil.wrapServletException(ServletUtil.java:61): ServletException including path '/WEB-INF/views/framework/page.jsp'.
at org.apache.jasper.el.JspValueExpression.getValue(JspValueExpression.java:123): /WEB-INF/tags/debug-principal.tag(19,1) '${PrincipalHelper.get()}' org.springframework.dao.DataAccessResourceFailureException: Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:255): Unable to acquire JDBC Connection; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48): Unable to acquire JDBC Connection
at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:695): HikariPool-1 - Connection is not available, request timed out after 30000ms.
at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
at sun.reflect.GeneratedConstructorAccessor70.newInstance(Unknown Source): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
WARN [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] SQL Error: 0, SQLState: 08S01
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] HikariPool-1 - Connection is not available, request timed out after 30001ms.
ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] HTTP GET /Starter-Project/WEB-INF/views/framework/page.jsp HTTP/1.1
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Status: 500
ERROR [org.apache.jsp.WEB_002dINF.views.framework.panic_jsp] Exception: org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467): Could not open JPA EntityManager for transaction; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection
org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48): Unable to acquire JDBC Connection
com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:695): HikariPool-1 - Connection is not available, request timed out after 30001ms.
com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
sun.reflect.GeneratedConstructorAccessor70.newInstance(Unknown Source): Communications link failure
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
java.net.DualStackPlainSocketImpl.waitForConnect(Native Method): Connection refused: connect
WARN [org.hibernate.orm.connections.pooling] HHH10001002: Using Hibernate built-in connection pool (not for production use!)
WARN [org.hibernate.orm.connections.pooling] HHH10001002: Using Hibernate built-in connection pool (not for production use!)
WARN [org.hibernate.orm.connections.pooling] HHH10001002: Using Hibernate built-in connection pool (not for production use!)
WARN [com.zaxxer.hikari.pool.HikariPool] {} - Thread starvation or clock leap detected (housekeeper delta={}).
